### DISCUSSION TITLE
How to install Keda using Yaml file based approach along with Pod identity?

### DISCUSSION THREAD

**Discussion by dc480506** [kedacore/keda#3034] [Q&A / Need Help] [answered]
For installing Keda with Pod identity, Helm chart provides a flag `--set podIdentity.activeDirectory.identity` to set the pod identity to be used for scalers but for the installation using Yaml file there's no documentation of how we can install with podIdentity. Any help or guidance would be really appreciated. Thanks!

**Comment by tomkerkhove** [comment, marked as answer]
In order for that to configure, we recommend going through the official pod identity documentation.

**Reply by dc480506** [reply]
Hey @tomkerkhove actually we have already setup the aad pod identity on our cluster. We've also created the `AzureIdentity` and `AzureIdentityBinding` for the keda pods which will have access to the scalers. We're finding it difficult to link this `AzureIdentityBinding` with the keda pods which is possible through Helm chart using `--set podIdentity.activeDirectory.identity` during Keda installation. But not understanding how to do the same linking if we install it via Yaml file approach.

**Reply by tomkerkhove** [reply]
I think all you need is https://github.com/kedacore/charts/blob/main/keda/templates/12-keda-deployment.yaml#L23-L25

**Reply by dc480506** [reply]
Thanks a lot @tomkerkhove  , I'll try doing this and get back to you if there are any issues I encounter.

**Reply by tomkerkhove** [reply]
Happy to help!
